docs/physics/update_and_conv/update_and_conv.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p><code>Things like this (where two things both affect each other on different ways) frequently occur in physics.</code></p><p>This is not really relevant to our documentation. I would remove it and rephrase the next sentence to</p><p> </p><p><code>We solve for the plasma properties by finding a steady-state</code></p><p> </p><p>and the last part of that paragraph to <code>have reached equilibrium.</code></p><p> </p><p>Second paragraph, replace "needed" with "necessary"</p><p> </p><p> </p><p>There's no point referencing "additional convergence strategies" if we don't have any</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>